src/material/core/ripple/ripple-renderer.ts

```diff
@@ -241,17 +237,31 @@ export class RippleRenderer {
     // Remove all previously registered event listeners from the trigger element.
     this._removeTriggerEvents();
 
-    this._ngZone.runOutsideAngular(() => {
-      this._triggerEvents.forEach((fn, type) => {
-        element.addEventListener(type, fn, passiveEventOptions);
-      });
-    });
-
     this._triggerElement = element;
+    this._registerEvents(pointerDownEvents);
+  }
+
+  /** Handles all registered events. */
```

Comment:
Could you add the `@docs-private` annotation to this JsDoc block? That will prevent it from appearing in our generated API docs.

Reply:
Done.